cardano-api/test/Test/Cardano/Api/Genesis.hs

```diff
@@ -22,19 +22,21 @@ import           Ouroboros.Consensus.Util.Time
 import           Cardano.Ledger.Address (Addr (..))
 import           Cardano.Ledger.Credential (Credential (..), PaymentCredential,
                    StakeCredential, StakeReference (..))
-import           Cardano.Ledger.BaseTypes (Network (..), truncateUnitInterval)
+import           Cardano.Ledger.BaseTypes (Network (..))
 import           Cardano.Ledger.Coin (Coin (..))
 import           Cardano.Ledger.Keys (GenDelegPair (..), Hash, KeyHash (..), KeyRole (..),
                    VerKeyVRF)
 import           Shelley.Spec.Ledger.PParams (PParams' (..), emptyPParams)
 
+import           Test.Shelley.Spec.Ledger.Utils (unsafeBoundRational)
+
 exampleShelleyGenesis :: ShelleyGenesis StandardShelley
 exampleShelleyGenesis =
   ShelleyGenesis
     { sgSystemStart = posixSecondsToUTCTime $ realToFrac (1234566789 :: Integer)
     , sgNetworkMagic = 4036000900
     , sgNetworkId = Testnet
-    , sgActiveSlotsCoeff = 6.259
+    , sgActiveSlotsCoeff = unsafeBoundRational (toRational @Double 6.259)
```

Comment:
`sgActiveSlotsCoeff` is now of type `PositiveUnitInterval`, which means it can only be in the range of `(0, 1]`. 

Therefore `6.259` is an invalid value and `unsafeBoundRational (toRational @Double 6.259)` will throw an error.

```suggestion
    , sgActiveSlotsCoeff = unsafeBoundRational 0.259
```

Reply:
Yes. I just noticed the same (I notice because it happens to fails to compile without `TypeApplications`, which are of course unnecessary). I happened to pick the same fix, 0.259 :smile:.